### PR TITLE
Enable persisting emUnits

### LINKED ISSUE #1025: Persist EmUnits
When persisting a JointGridContainer to csv, the EmUnits should also be persisted.

### PR DESCRIPTION
**Pull request by jo-bao**
Resolves #1025

### CODE REVIEW

**Comment by jo-bao** [comment]
!test

**Comment by t-ober** [comment]
!test

**Comment by t-ober** [comment]
!test

File: src/main/java/edu/ie3/datamodel/io/extractor/HasEm.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+*/
+package edu.ie3.datamodel.io.extractor;
+
+import edu.ie3.datamodel.models.input.EmInput;
+import java.util.Optional;
+
+/**
+ * Interface that should be implemented by all elements holding one or more {@link
```
Comment (sebastian-peter) [inline src/main/java/edu/ie3/datamodel/io/extractor/HasEm.java:12]: "one or more" is not correct here

**Review by sebastian-peter — CHANGES_REQUESTED** [review]

File: src/main/java/edu/ie3/datamodel/models/input/EmInput.java [hunk 2]
```diff
@@ -92,15 +93,20 @@ public String toString() {
...
         + ", id='"
         + getId()
-        + ", operator="
+        + "', operator="
         + getOperator().getUuid()
         + ", operationTime="
         + getOperationTime()
         + ", controlStrategy="
         + getControlStrategy()
         + ", parentEm="
```
[the remark this answers is not in the document]
Comment (jo-bao) [reply src/main/java/edu/ie3/datamodel/models/input/EmInput.java:102]: Yes!

File: src/main/java/edu/ie3/datamodel/models/input/EmInput.java [hunk 3]
```diff
@@ -100,13 +96,13 @@ public String toString() {
...
```
Comment (sebastian-peter) [inline src/main/java/edu/ie3/datamodel/models/input/EmInput.java:98]: Don't forget to adapt `toString` here as well 😎

**Review by sebastian-peter — CHANGES_REQUESTED** [review]

**Pull request merged**